app/views/spree/admin/overview/single_enterprise_dashboard.html.haml

```diff
@@ -47,14 +47,15 @@
     .header
       %h3
         %span.icon-map-marker
-        Your profile live
-      %p on the Open Food Network map
+          = t "your_profil_live"
+      %p
+        = t "on_ofn_map"
     .list
       /-# Can we pass an anchor here to zoom to our enterprise?
       %a.button.bottom{href: main_app.map_path, target: '_blank'}
-        See
+        = t "see"
```

Comment:
Take a look on this:http://stackoverflow.com/questions/10948333/how-to-use-rails-i18n-with-haml
Unfortunatly, the link is broken, but I suppose the answer was right so I propose 
t 'activerecord.errors.messages.record_invalid' as model.